app/styles/homepage.scss

```diff
@@ -0,0 +1,8 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+.standard-link-container {
+    padding: 15px;
+    @apply basis-1/3;
```

Comment:
Basis 1/3 doesn't work super well because of the text wrapping 😧 
<img width="437" alt="image" src="https://github.com/jjm3x3/AmazingRaceFantasy/assets/5040219/69f12d65-bdf9-415d-8b67-b4ce8d79fde5">

Reply:
Updated

<img width="525" alt="image" src="https://github.com/jjm3x3/AmazingRaceFantasy/assets/9656794/5609d312-3b61-45b7-97dc-b800842dacfc">